mock-core-configs/etc/mock/templates/centos-stream-10.tpl

```diff
@@ -1,6 +1,7 @@
 config_opts['chroot_setup_cmd'] = 'install bash bzip2 centos-stream-release coreutils cpio diffutils findutils gawk glibc-minimal-langpack grep gzip info patch redhat-rpm-config rpm-build sed shadow-utils tar unzip util-linux which xz'
 config_opts['dist'] = 'el10'  # only useful for --resultdir variable subst
 config_opts['releasever'] = '10'
+config_opts['releasever_major'] = '10'
```

Comment:
This doesn't do anything. There's nothing wired to this option in Mock.

Reply:
I could adjust the config to only use `releasever`, I was just trying to be explicit about the places that are definitely always just the major version, even if someone tries to use the template on a distro that sets `releasever` to a full version such as 10.0.  What do you think @praiskup?  If it's preferable to stick with `releasever_major` I can send a separate PR to "wire it up" as Neal described (but I could use a pointer to where in the code this happens).

Reply:
`I would certainly prefer it to be wired up (alongside `releasever_minor`)  once that becomes useful.

Reply:
I would certainly prefer it to be wired up (alongside `releasever_minor`)  once that becomes useful.

Reply:
By defining `releasever_major = 10` in this template you wire it up; then just use the `{{ jinja }}` statement to read that value.

Reply:
So if I'm understanding you right, `config_opts['releasever']` doesn't actually correspond to `$releasever` in templates?  I can see how https://github.com/rpm-software-management/mock/blob/main/mock-core-configs/etc/mock/templates/centos-stream-9.tpl uses `{{ releasever }}`, so I can do the same style.

Reply:
Would wiring it up properly just involve adding `releasever_major` to https://github.com/rpm-software-management/mock/blob/main/mock/py/mockbuild/package_manager.py#L455-L458 ?

Reply:
Good catch there; that mechanism though probably only affects bootstrap at this point in time (and some users still --no-bootstrap-chroot is still supported).  I am not saying that we shouldn't change mock so this substitution works everywhere, but the change would mean new release of mock (not just configs), and probably a dedicated separate pull-request.

Reply:
Understood, I'll start with the jinja style for now, and look into the other way in the future.  Thanks for the clarification.